raiden/network/proxies/token_network.py

```diff
@@ -1149,6 +1150,7 @@ def set_total_withdraw(
                     partner=participant,
                     block_identifier=given_block_identifier,
                 )
+                current_block_number = self.client.get_block(given_block_identifier).number
```

Comment:
```suggestion
                given_block_number = self.client.get_block(given_block_identifier)["number"]
```

1. Convention uses `given_block_number` as the name (because it doesn't necessarily have to be the current
2. `get_block` returns a dict